keras/export/export_lib.py

```diff
@@ -203,6 +206,15 @@ def add_endpoint(self, name, fn, input_signature=None):
                 per positional input argument of `fn`). Nested arguments are
                 allowed (see below for an example showing a Functional model
                 with 2 input arguments).
+            alias: Str, an alias for the `tf.function` created for `fn`, which
```

Comment:
Is there necessarily at most 1 alias per endpoint? Also, can we provide a description that answers "what does this represent and when should I use it"? The current description defers to the TF docs, but that part of the docs is extremely cryptic.

Reply:
Removed argument. Instead, the `tf.function` is returned, which can be used to add aliases.